dtfs-central-api/src/services/state-machines/fee-record/event-handlers/add-a-payment/add-a-payment.event-handler.ts

```diff
@@ -0,0 +1,18 @@
+import { FeeRecordEntity } from '@ukef/dtfs2-common';
+import { NotImplementedError } from '../../../../../errors';
+import { BaseFeeRecordEvent } from '../../event/base-fee-record.event';
+
+type AddAPaymentEventPayload = {
+  paymentId: number;
+};
+
+export type FeeRecordAddAPaymentEvent = BaseFeeRecordEvent<'ADD_A_PAYMENT', AddAPaymentEventPayload>;
+
+export const handleFeeRecordAddAPaymentEvent = (
+  /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Can we TO DO removing these eslint-disables against the ticket number that is going to actually handle this payment event? Of the top of my head that might be [FN-1739](https://ukef-dtfs.atlassian.net/browse/FN-1739)

Reply:
I can't find a more appropriate ticket to attach this todo to, so I'll use this one